tests/models/test_model.py

```diff
@@ -128,17 +158,27 @@ def test_model_info():
             model_info = Model.log(
                 "some/path", TestFlavor, signature=sig, input_example=input_example
             )
-        local_path = _download_artifact_from_uri(
-            "runs:/{}/some/path".format(run.info.run_id), output_path=tmp.path("")
-        )
+        model_uri = "runs:/{}/some/path".format(run.info.run_id)
 
-        assert model_info.run_id == run.info.run_id
-        assert model_info.artifact_path == "some/path"
-        assert model_info.model_uri == "runs:/{}/some/path".format(run.info.run_id)
+        with warnings.catch_warnings(record=True) as w:
+            model_info_fetched = mlflow.models.get_model_info(model_uri)
+            assert (
+                "Field signature_dict is deprecated since v1.28.1. Use signature instead."
+                in str(w[0].message)
+            )
+        local_path = _download_artifact_from_uri(model_uri, output_path=tmp.path(""))
+
+        assert model_info.run_id == run.info.run_id == model_info_fetched.run_id
```

Comment:
```suggestion
        assert model_info.run_id == run.info.run_id
        assert model_info_fetched.run_id == run.info.run_id
```

Can we split the comparison to make it easier to debug failures?